packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -1932,8 +1933,29 @@ contract OPContractsManager is ISemver {
             thisOPCM.setRC(false);
         }
 
+        // Get the superchain config and superchain's proxy admin via the system config.
+        ISuperchainConfig _superchainConfig = superchainConfig;
+        IProxyAdmin _superchainProxyAdmin = superchainProxyAdmin;
+        if (_opChainConfigs.length > 0) {
+            // Ideally use _opChainConfigs[0].systemConfigProxy.superchainConfig()
```

Comment:
Why does the comment say "Ideally"? It suggests to me that there is a failure mode that we're going to handle.